content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md

```diff
@@ -0,0 +1,153 @@
+
+
+
+# Deploy CockroachDB
+
+Palette users can deploy CockroachDB to a virtual cluster by using the following steps.
+
+## Prerequisite
+
+- A Spectro Cloud [account](https://www.spectrocloud.com/get-started/).
+
+
+- A Virtual Cluster with at least the following minimum resources.
+    - 8 CPU 
+    - 8 GB of Memory
+    - 8 GB of Storage.
+
+
+## Enablement
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+
+2. On the right side of the window, click on the **User Menu** and select **Switch to App Mode**.
+
+
+
+3. Navigate to the left **Main Menu** and click on **App Profiles** to create a [new App Profile](/devx/app-profile/create-app-profile/). Provide the following basic information and click **Next**.
+
+|         Parameter           | Description  |
+|-----------------------------|-----------------|
+|Application Profile Name | A custom name for the app profile.|
+|Version (optional)       | The default value is 1.0.0. You can create multiple versions of an app profile using the format **`major.minor.patch`**.
+|Description (optional)   | Description of the app profile. | 
+|Tag (optional)           |  Assign tags to the app profile.|
+ 
+
+4. Select the **CockroachDB** service and start the configuration.
+  
+
+
+5. Provide the following information to the wizard:
+  * **Name**: The application name.
+
+  * **Username**: The user name for database access control.
+
+
+  * **dbPassword**: Security password for the DB service.
+
+
+  * **Database Name**: The name of the database to target.
+
+
+  * **PersistentVolumeClaim Size (GiB)**: Select the volume as per the storage volume available in the cluster group and virtual clusters. Ensure you do not exceed the maximum storage size for your virtual cluster. 
+
+
+6. Save your changes.
+
+
+7. Deploy the app profile to a Palette Virtual Cluster. Use the [Deploy a Virtual Cluster](/clusters/palette-virtual-clusters/deploy-virtual-cluster#deployavirtualcluster) guide for additional guidance or check out the [Deploy an Application using Palette Dev Engine](/devx/apps/deploy-app) tutorial.
+
+
+## Validation
+
+* To verify your database service is in the app profile, navigate to the **App Profiles** page, where all your app profiles are listed. Select the app profile to review the service layers. The following screen displays the different service layers that make up the app profile. Ensure CockroachDB is an available service layer.
+
+
+* Validate the services from the App page after app deployment. First, navigate to the **Apps** page, where all your apps are listed. Then, select the app to display the service layers. The color code in the app profile box shows the status of the service deployment.
+
```

Comment:
I'm okay with using this Validation format, but I thought you wanted numbered steps?

Reply:
You're absolutely right. I forgot to push it up my last change